codegen/projections/weather/lib/weather/waiters.rb

```diff
@@ -78,7 +78,7 @@ def initialize(client, options = {})
       attr_reader :tags
 
       # @param (see Client#get_city)
-      # @return [true, Hearth::Waiters::WaiterFailed]
+      # @return [true]
```

Comment:
Maybe I've forgotten how our waiters work.  Do they raise an exception instead of returning false? IF so, maybe that should be documented on the wait method?

Reply:
They raise an exception or return true. So this part was not correct (encountered this trying to test the Singleton method change). I think I can add a raises yard tag here.